### PR TITLE
feat: add default focus color token

### PR DESCRIPTION
**Pull request by alisonailea**
**Related Issue:** #7180 #10510

## Summary

This adds `calcite-color-focus` to replace the deprecated `calcite-ui-focus-color`

### CODE REVIEW

**Review by geospatialem — COMMENTED** [review]
@alisonailea Awesome! 💨 Can you also add in the more related #10510 to the issue context?

**Review by jcfranco — CHANGES_REQUESTED** [review]
Do we need to generate separate `semantic` output for the default focus color to be added? This change is breaking as it removes exports from the `global` files (causing [this build failure](https://github.com/Esri/calcite-design-system/actions/runs/11336635809/job/31526876046?pr=10512#step:5:608)).

File: packages/calcite-design-tokens/src/$config.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export const config: CalciteTokenTransformConfig = {
         breakpoint: "breakpoints.js",
         index: {
           name: "index.js",
-          export: ["./global.js", "./breakpoints.js"],
+          export: ["./global.js", "./semantic.js", , "./breakpoints.js"],
```
Comment (jcfranco) [inline packages/calcite-design-tokens/src/$config.ts:54]: Should've caught this sooner, but there's an extra `,` here.

File: packages/calcite-design-tokens/support/tests/__snapshots__/index.spec.ts.snap [hunk 2]
```diff
@@ -840,431 +745,14 @@ export const calciteZIndex9 = "900";
 exports[`generated tokens ES6 global should match 1`] = `
 "/
 
-export const calciteBorderWidthNone = "0";
-export const calciteBorderWidthSm = "1px";
-export const calciteBorderWidthMd = "2px";
-export const calciteBorderWidthLg = "4px";
-export const calciteColorBackground = {"light":"#f8f8f8","dark":"#353535"};
-export const calciteColorBackgroundNone = "rgba(255, 255, 255, 0)";
```
Comment (jcfranco) [inline packages/calcite-design-tokens/support/tests/__snapshots__/index.spec.ts.snap:848]: Also, can you update the snapshot to cover semantic tokens? These are no longer present.

**Review by jcfranco — APPROVED** [review]
👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾
👾🪙👾👾👾🪙👾👾🪙🪙👾👾🪙🪙🪙👾👾🪙🪙🪙👾🪙🪙🪙🪙👾🪙👾
👾🪙🪙👾👾🪙👾🪙👾👾🪙👾👾🪙👾👾🪙👾👾👾👾🪙👾👾👾👾🪙👾
👾🪙👾🪙👾🪙👾🪙👾👾🪙👾👾🪙👾👾🪙👾👾👾👾🪙🪙🪙👾👾🪙👾
👾🪙👾👾🪙🪙👾🪙👾👾🪙👾👾🪙👾👾🪙👾👾👾👾🪙👾👾👾👾👾👾
👾🪙👾👾👾🪙👾👾🪙🪙👾👾🪙🪙🪙👾👾🪙🪙🪙👾🪙🪙🪙🪙👾🪙👾
👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾👾

**Pull request merged**